tvb_library/tvb/basic/neotraits/info.py

```diff
@@ -94,23 +95,35 @@ def narray_summary_info(ar, ar_name='', omit_shape=False):
         return {'is None': 'True'}
 
     ret = {}
-    if not omit_shape:
-        ret.update({'shape': str(ar.shape), 'dtype': str(ar.dtype)})
 
-    if ar.size == 0:
-        ret['is empty'] = 'True'
+    if condensed_form:
+        condensed_desc = ""
+        if ar.shape == (1,):
```

Comment:
I think for condensed form also `is empty` or `has NaN` would be interesting to display.
What about `arr = None ` ?  NPE ?

Reply:
I think None case is not solved. Is it ?